ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAPP-672 Update URLs

## Description of change

Update content for displayed URLS in routes file

## Link to relevant ticket

[CRIMAPP-672](https://dsdmoj.atlassian.net/browse/CRIMAPP-672)

## Notes for reviewer
N.B. '/which_benefits' is incorrect in spreadsheet and if it hasn't been updated needs to be '/which_benefits_client' - Arthur (have notified Rachel)
## Screenshots of changes (if applicable)

### Before changes:

### After changes:

## How to manually test the feature

For each URL highlighted in yellow in [this spreadsheet](https://docs.google.com/spreadsheets/d/1gdDpuyqWO6MJgUT1SOy0nbHd8AAdp1YebBzh1WDG040/edit#gid=0 )
navigate to the page and check that the URL matches the one in the spreadsheet


[CRIMAPP-672]: https://dsdmoj.atlassian.net/browse/CRIMAPP-672?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
